packages/sdk-metrics/src/Descriptor.ts

```diff
@@ -32,32 +32,64 @@ export enum InstrumentType {
 /**
  * An interface describing the instrument.
  */
-export interface InstrumentDescriptor {
+export interface Descriptor {
   readonly name: string;
   readonly description: string;
   readonly unit: string;
-  readonly type: InstrumentType;
   readonly valueType: ValueType;
 }
 
-export function createInstrumentDescriptor(
+/**
+ * An interface describing the instrument.
+ * @deprecated Please use {@link Descriptor} instead.
+ */
+export interface InstrumentDescriptor extends Descriptor {
+  /**
```

Comment:
Following from https://github.com/open-telemetry/opentelemetry-js/issues/3439#issuecomment-1334958746, can you add a note in the comment mentioning `MetricData.dataPointType` as recommended alternative?

Reply:
Added a note to reference the data point type in [014c7f1](https://github.com/open-telemetry/opentelemetry-js/pull/3520/commits/014c7f15056c33dc2516e1f400d6f25bf0ad43fa) :slightly_smiling_face: